src/utils/keyboard.ts

```
@@ -69,10 +69,33 @@ export const FOCUSABLE_SELECTOR =
+  // This is necessary because some browser implementations (like jsdom) may return
+  // elements in selector order rather than document order
+  elementsArray.sort((a, b) => {
+    const position = a.compareDocumentPosition(b)
+
+    // Handle containment relationships (one node contains the other)
+    if (position & Node.DOCUMENT_POSITION_CONTAINED_BY) {
+      return 1 // b contains a, so b comes first
+    } else if (position & Node.DOCUMENT_POSITION_CONTAINS) {
+      return -1 // a contains b, so a comes first
+    } else if (position & Node.DOCUMENT_POSITION_FOLLOWING) {
+    } else if (position & Node.DOCUMENT_POSITION_FOLLOWING) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)